server/controllers/user.js

```diff
@@ -77,6 +77,54 @@ function queryCAS(login, pass) {
  * @apiSuccess {number} avatar.mouth      Number of the mouth
  */
 
+/**
+ * @api {post} /users/ Get data of several users
```

Comment:
Why is it a `post` endpoint?

Reply:
The users whose information you wish to obtain are in an array passed by POST (as in the APIs given as an example).

Reply:
It's true, the example API use this method. I'm wondering if we can use a `GET` endpoint with query parameters for the ids, as we are not likely to ask for a big number of users per request. I'm a bit bothered to use a `POST` endpoint to get data.

Reply:
> as we are not likely to ask for a big number of users per request
I disagree, the user can have have as many challenges as he wants. We could request 2 users as 20, or more.

Reply:
Hum, you're right. Let's go for a `POST` endpoint :+1: